### PR TITLE
reformat tile

### PR DESCRIPTION
**Pull request by daisy-dotcom**
- [ ] #2078

### CODE REVIEW

**Review by MarShaikh — CHANGES_REQUESTED** [review]
Hi, thanks for the examples. They look good but could you also add Array and container instance examples? Plus, can you also look at the backends, I noticed that some of them need to be reformatted as well! Thanks!

**Comment by daisy-dotcom** [comment]
@MarShaikh will get to it as soon as i can (:

File: ivy/functional/ivy/manipulation.py [hunk 1]
```diff
@@ -1146,6 +1146,86 @@ def tile(
...
+    >>> x = ivy.Container( a = ivy.array([0,1,2]), b = ivy.array([[3],[4]]))
+    >>> y = ivy.tile(x, (2))
+    >>> print(y)
+    {
+        a = [0,1,2,0,1,2]
+        b = [[3,3],
+             [4,4]]
+    }
+
+    Instance Method Examples
```
Comment (MarShaikh) [inline ivy/functional/ivy/manipulation.py:1200]: The instance methods example belong to the `tile` instances in the `ivy/array` and `ivy/container`.

**Review by MarShaikh — CHANGES_REQUESTED** [review]
Everything else looks good. Please just remove the instance methods from the `ivy/functional` and add them to `tile` functions's array and container instances. Thanks!

**Comment by daisy-dotcom** [comment]
@MarShaikh okay. i moved them to the correct directories

**Review by MarShaikh — APPROVED** [review]
Lgtm!

**Review by MarShaikh — CHANGES_REQUESTED** [review]
Everything else in the code looks good, but you still need fix the docstring errors and the linting error. For the lint error, please just set up `pre-commit`, it will makes things a lot easier. You find more info on this in our docs. Thanks.

**Comment by daisy-dotcom** [comment]
Hi @MarShaikh . I've fixed the lint error but I keep failing the docstring test. I checked and it says it's the depthwise_conv2d method with the issue. Is that okay?

**Comment by MarShaikh** [comment]
> Hi @MarShaikh . I've fixed the lint error but I keep failing the docstring test. I checked and it says it's the depthwise_conv2d method with the issue. Is that okay?

Yes, since it isn't related to your function it is fine! I'll review your PR now.

**Review by MarShaikh — APPROVED** [review]
LGTM!

**Comment by daisy-dotcom** [comment]
Thank you @MarShaikh

**Pull request merged**